### PR TITLE
mc_module

### PR DESCRIPTION
**Pull request by yohanna123**

### CODE REVIEW

File: quest/core/mc_functions.cpp [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+                }
+       }
+  }
+
+  return e_pair;
+}
+
+PYBIND11_PLUGIN(mc_demo)
+{
+   py::module m("mc_demo", "Dibyendu's basic module");
```
Comment (janash) [inline quest/core/mc_functions.cpp]: Can we have more descriptive names here? This is no longer an MC "demo", and description should match module (not "basic module")

**Comment by yohanna123** [comment]
We have changed the module

**Comment by dgasmith** [comment]
Looks great, can we add a test for this in the `/tests` folder? You can get reproducible random results by using the `np.random.seed(0)` command.

**Comment by dgasmith** [comment]
@yohanna123 Will you be available to add a small test?

**Comment by dgasmith** [comment]
Going to merge this in and fix up the other MC parts.

**Comment by yohanna123** [comment]
Hi @dgasmith , yes, I plan to. I'm still working on making the function work though.

**Pull request merged**